chef/cookbooks/provisioner/templates/default/crowbarrc.erb

```diff
@@ -0,0 +1,7 @@
+[default]
+server = <%= @server %>
+username = crowbar
```

Comment:
Does this **have to be** `crowbar`?

Reply:
I guess not, that could be changed here https://github.com/crowbar/crowbar-core/blob/master/chef/data_bags/crowbar/template-crowbar.json#L21

So this should probably be smarter.